maestro-cli/src/main/java/maestro/cli/util/ErrorReporter.kt

```diff
@@ -0,0 +1,43 @@
+package maestro.cli.util
+
+import maestro.cli.api.ApiClient
+import picocli.CommandLine
+import java.security.MessageDigest
+import java.util.concurrent.Executors
+import java.util.concurrent.TimeUnit
+import kotlin.Exception
+
+object ErrorReporter {
+
+    private const val BASE_API_URL = "https://api.mobile.dev"
```

Comment:
Nit: we're defining `BASE_API_URL` here and also in `Updates.kt`. Shall we define it once somewhere and use it everywhere instead?